crates/orderbook/openapi.yml

```diff
@@ -1448,3 +1557,39 @@ components:
         totalSurplus:
           type: string
           description: The total surplus.
+    InteractionData:
+      type: object
+      properties:
+        target:
+          $ref: "#/components/schemas/Address"
+        value:
+          type: number
```

Comment:
Since this is a `U256` it should probably be a decimal string.

Reply:
Right, fixed!